chain/rust/src/builders/mod.rs

```diff
@@ -0,0 +1,9 @@
+pub mod certificate_builder;
+pub mod input_builder;
+pub mod redeemer_builder;
+pub mod output_builder;
+//pub mod tx_builder;
```

Comment:
is this meant to be commented out?

Reply:
For now, yes. The rest of the builders all compile since they've been updated. The tx_builder.rs does not yet. I will uncomment this once that's ready. It's a draft PR still.